english/Enter-a-dim-reality/continue-forward/Forward_Stay/forwardStay.md

```diff
@@ -1,4 +1,6 @@
 You stay on the bank, unsure of what trouble might await you in the water.
 You sit down, mesmerized by the light.  It begins to fade - you begin to panic.  
-The light is gone, you scan the horizon frantically, desperately hoping to see 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.